src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt

```diff
@@ -165,9 +166,9 @@ class DataCenterFormula private constructor(
 
         val machines = jiraStack.listMachines()
         val jiraNodes = machines.filter { it.tags.contains(Tag("jpt-jira", "true")) }
-        val databaseIp = machines.single { it.tags.contains(Tag("jpt-database", "true")) }.publicIpAddress
+        val databaseIp = InstanceAddressSelector.getReachableIpAddress(machines.single { it.tags.contains(Tag("jpt-database", "true")) })
```

Comment:
Stylistic tip:
If you want to avoid additional bracket-nesting, you can:
```suggestion
        val databaseIp = machines.single { it.tags.contains(Tag("jpt-database", "true")) }.let { InstanceAddressSelector.getReachableIpAddress(it) }
```

Reply:
I've extracted the tag handling to a separate file instead